examples/todo-app/package.json

```diff
@@ -16,7 +16,8 @@
     "@prodo/logger": "^0.0.9",
     "core-js": "2",
     "react": "^16.8.6",
-    "react-dom": "^16.9.0"
+    "react-dom": "^16.9.0",
+    "styled-components": "^4.4.0"
```

Comment:
If the versions are different does it break?

Reply:
Actually, it seems to be fine now. I thought that was causing issues initially but I think it was mostly the types + that last commit. (If you want me to change the docs or todo-app back to 4.3.2 or something, lmk)

Reply:
This version is okay. I was just didn't think we should be forcing users to use a specific version of styled-components. If multiple versions are allowed then that is good 👍

Reply:
I do remember having a bunch of issues with `@types/styled-components`